### PR TITLE
[PUB-2571] Queues render Post directly

### PR DESCRIPTION
**Pull request by anafilipadealmeida**
## Description
- Queues render Post directly
- Fix Lightbox bug

Step 5 of Posts refactoring: https://paper.dropbox.com/doc/Refactor-Posts--AyD9S0MqLq3mO5bw~~BV9TzyAg-rgzLttQL5bKmyBrrlvAgn
## Context & Notes
Next PR: delete TextPost, ImagePost, MultipleImagePost, VideoPost and LinkPost
https://buffer.atlassian.net/browse/PUB-2571

## Checklist

-   [ ] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] I have tested different user stories. 
-   [ ] I have identified similar or related features and tested they are still working.
-   [ ] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/queue/components/QueueItems/index.jsx [hunk 1]
```diff
@@ -9,11 +9,7 @@ import {
   transitionAnimationType,
 } from '@bufferapp/components/style/animation';
 import {
-  TextPost,
-  ImagePost,
-  MultipleImagesPost,
-  LinkPost,
-  VideoPost,
+  Post,
```
Comment (karelm) [inline packages/queue/components/QueueItems/index.jsx:12]: 😍

File: packages/shared-components/PostList/index.jsx [hunk 2]
```diff
@@ -35,11 +31,11 @@ const HeaderStyle = styled.div`
...
-  ['text', TextPost],
-  ['image', ImagePost],
-  ['multipleImage', MultipleImagesPost],
-  ['link', LinkPost],
-  ['video', VideoPost],
+  ['text', Post],
+  ['image', Post],
+  ['multipleImage', Post],
+  ['link', Post],
+  ['video', Post],
```
Comment (hamstu) [inline packages/shared-components/PostList/index.jsx:38]: #WeAreAllPost

File: packages/shared-components/PostList/index.jsx [hunk 3]
```diff
@@ -92,7 +92,7 @@ const renderPost = ({
     profileServiceType,
   };
   let PostComponent = postTypeComponentMap.get(post.type);
-  PostComponent = PostComponent || TextPost;
+  PostComponent = PostComponent || Post;
```
Comment (karelm) [inline packages/shared-components/PostList/index.jsx:95]: can this not just be replaced with (all good if you are still in progress and this is part of step5/6 😉)
```suggestion
  PostComponent = Post;
```

Comment (anafilipadealmeida) [reply packages/shared-components/PostList/index.jsx:95]: This one still has Story variation, I'll make it look nicer in a future PR!

**Pull request merged**